src/main/webapp/js/userPage.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+const h = React.createElement;
+
+class UserPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { input: "" }; //user's input in about me container
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  fetchData() {
+    const url = `/about?user=${this.props.userEmail}`;
+    fetch(url)
+      .then(res => {
+        return res.text();
+      })
+      .then(currentValue => {
+        this.setState({ currentValue });
+      });
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  render() {
+    // pure function
+    const button = h(
+      "button",
+      {
+        onClick: () =>
+          fetch("/about", {
+            method: "POST",
+            credentials: "same-origin",
+            headers: { "Content-Type": "application/x-www-form-urlencoded" },
+            body: `about-me=${this.state.input}`
```

Comment:
You should use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent here. Currently this will fail if input is something like `input&name=whatever`.